airflow/providers/jenkins/sensors/jenkins.py

```diff
@@ -0,0 +1,64 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from typing import Any, Dict, Optional
+from urllib.error import HTTPError, URLError
+
+import jenkins
+from jenkins import JenkinsException
+
+from airflow.exceptions import AirflowException
+from airflow.providers.jenkins.hooks.jenkins import JenkinsHook
+from airflow.sensors.base import BaseSensorOperator
+
+class JenkinsSensor(BaseSensorOperator):
```

Comment:
Should we rename it to `JenkinsBuildSensor`?
This may be more explicit and will be helpful for cases where we will have other sensor in Jenkins (if that make sense)?

Reply:
That is actually makes a lot of sense, perfect naming for the purpose 
applied